db/migrate/20230921004930_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, null: false, foreign_key: true
+      t.references :tweet, null: false, foreign_key: true
```

Comment:
First you need to create your tweets table before creating other table that points to it

```
aused by:
PG::UndefinedTable: ERROR:  relation "tweets" does not exist
/home/dev0/projects/ror_training_project/week2/francisco-rodriguez/db/migrate/20230921004930_create_likes.rb:3:in `change'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)

```